### ISSUE TITLE
Not Clear How to Get the Trial?

### ISSUE THREAD

**Issue by spiritfly** [MicrosoftDocs/OfficeDocs-SkypeForBusiness#4307]
This is not clear on how to get the trial. I tried contacting the representative as said here, it's been two days and I still haven't received a response. 

---
#### Document Details

⚠ *Do not edit this section. It is required for docs.microsoft.com ➟ GitHub issue linking.*

* ID: 779619a7-95a8-c775-0538-4f07abb7067e
* Version Independent ID: cf930116-3d9d-2075-5291-f819cb17a234
* Content: [Manage the free Office 365 E1 Trial - Microsoft Teams](https://docs.microsoft.com/en-us/microsoftteams/e1-trial-license#feedback)
* Content Source: [Teams/e1-trial-license.md](https://github.com/MicrosoftDocs/OfficeDocs-SkypeForBusiness/blob/live/Teams/e1-trial-license.md)
* Service: **msteams**
* GitHub Login: @LolaJacobsen
* Microsoft Alias: **lolaj**

**Comment by Biobobski** [comment]
As a Microsoft Partner this has been a pain, 3 of my suppliers in the UK emailed the offer but none have managed after 3 days and 2 support logs with Microsoft. It is not helping anyone just adding to my workload. Can we get this sorted out as I have customers who would like this but 3+ days each it cannot be done

**Comment by Biobobski** [comment]
Finally got the code after 3 days, went to apply it and got the following “ Sorry

The service you’re subscribing to isn’t available for one of the following reasons:
It’s no longer offered by Microsoft.
You’ve already tried this service. You can sign up for a trial only once.
The beta or trial you’re requesting has expired.
The promotion code has already been used.
For more information, contact the Microsoft partner or Microsoft sales agent that gave you the link to this offer.”

I am not going to bother for any other customers if I get this one done, it is far too time consuming and I have not even got to the setup part

**Comment by ronbass** [comment]
I spent half a day searching for instructions and clues about adding the free trial for Office 365 E1, I can only see it is $10+ per month per user and there is nothing free about that.
It is ridiculous to make admins waste so much time which has more value than the $10 per month at this point. Real shame that even guys who intervene here to help are asking us to open miserable tickets that will lead nowhere, more waste of time really.
You didn't need to propose anything but when you do please go the end of it. Thank you for your efforts.

**Comment by Biobobski** [comment]
The url was sent with the promo code embedded after 3 days of back and forth from Microsoft via my supplier, i followed the instructions on a blank laptop with no accounts on with private browsing and it failed.

**Comment by Biobobski** [comment]
I pasted the url i received into a google search as it fails no matter what without even trying to signup or open a new account and it matched an open ticket for a trial, the codes are not unique so even more weird

**Comment by spiritfly** [comment]
I opened a service request asking for this trial. We really could not commit to proper deployment because of moving everyone from home, so we couldn't test Microsoft Teams really well. So first a girl representative replied to that ticket after 3 days. For some reason, I am not getting emails delivered whenever someone replies to the service request. Apparently it is the only way to reply to the service request ticket. So I manually entered the support request as a title in an email and replied. The representative was trying to reach me on my phone, despite me choosing Email as the preferred way of communication when creating the service request. The first day, she called during off office hours so I couldn't pick up. The second day she called at the end of the day. I had a REALLY hard time understanding her because the line was cutting out. It was like a bad VoIP call or something. So after a lot of talks she barely understood what I needed even though I clearly said in the support request that I want to try out the new E1 6-month offer. Then she put me on hold for like 5-10 minutes. I patiently waited out, but she said she would have to discuss this with her internal team and she would have to call me on Monday(it was Friday when she called).

So all in all a lot of wasted time both for Microsoft's support and me with no result at the end. I have a feeling that they won't even approve the offer at the end, but let's see.

**Comment by ronbass** [comment]
Most of customers support centers are located in Asia, like Philippines or India (not just MS but many big Telecom and Software Cos.), generally they are opened 24h then when they need to escalate an issue to level 2 or 3, it will get transferred to America or Europe, so it takes time. Because of low budgets, the poor guys trying to help at level 1 are not sufficiently trained to respond adequately or solve the issues in a timely manner.
Callers often get upset because VOIP lines are not clear, accents sometimes are hard to understand but mainly because customers know more than the support guys and tend to lose their patience with them, on the other side many don't understand what is the issue about. Sorry about bringing that reality. Patience is important here.

**Comment by Biobobski** [comment]
It should need this level of involvement from any support, Microsoft should have had the offer ready to go simple and easily like any other free trial. If you get a trial of spotify you dont spend 4 days trying to get the details and then get sent a generic repeated code that doesnt even work

**Comment by scanum** [comment]
@spiritfly Thank you for submitting feedback and contributing to the docs. We are currently investigating this.

**Comment by ronbass** [comment]
So far I don't see any resolution, every day that goes by is just discouraging, why make it simple when we can make it complicated?
How many days or weeks do you need to investigate that, is your team communicating through pipes? Does it really need an investigation when the offer was clear?

**Comment by Reeced40** [comment]
@LolaJacobsen please can you assist.

**Comment by LolaJacobsen** [comment]
As of July 1, 2020, the Office 365 E1 Trial license is no longer available.
